### ISSUE TITLE
yun.ir nuisance (timer)

### ISSUE THREAD

**Issue by MasterKia** [uBlockOrigin/uAssets#13610]
### Prerequisites

- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues) to avoid opening a duplicate issue
- [X] I am running the latest version of uBlock Origin

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### URL(s) where the issue occurs

`https://yun.ir/hj6nr6`

### Describe the issue

`yun.ir` is a link shortener.

When visiting `https://yun.ir/hj6nr6`, I need to wait 25 seconds to get to the destination URL:
`<meta http-equiv="Refresh" content="25; url='https://docs.google.com/forms/d/e/...."/>`
But if I visit it again because there is a `verified` cookie I don't have to wait and it instantly redirects me to the destination URL.

I think the cookie check is server-side, so no bypass there.

Is it possible to bypass the timer?
`refresh-defuser` would break the redirect, and `nano-stb`/`nano-sib` wouldn't work.

### uBlock Origin version

Latest

### Browser name and version

Chrome

### Settings

-

**Comment by krystian3w** [comment]
In AdGuard you can bake that cookie to exclude server side checking and add this to your AdGuard supplement file if cover PersianList.

Start from `set-cookie` and if baked too late then use `set-cookie-reload`.

**Comment by mapx-** [comment]
uBO does not handle the cookies, I don't see other way

**Comment by krystian3w** [comment]
But webmaster may found this issue or your list and for AG users enforced check values.

Then are two ways - pure JS rule in AG Base or AG annoyance (after detect many visitors) or additional supplement with recommended check "trusted filter" (in AG syntax filter means list/file with own rules) in AdGuard subscribe custom list mechanism.

**Comment by MasterKia** [comment]
> webmaster may find this issue/your list and enforce check value for AG users.

True, but I didn't modify _their_ cookie; the first time you visit one of their shortened links, they create a "verified" cookie so that in your next visit you won't have to wait 25 seconds.
My solution: I just added a dummy cookie using AG `set-cookie-reload` which causes the site to reload once, and since the "verified" cookie is already created on the first visit, then after the reload site doesn't enable a timer.

Of course when `yun.ir` finds out about this, it _could_ counter my bypass.

> * rule in AG Base or AG annoyance (after detect many visitors)

That's a good idea, `yun.ir` has 337 ranking in Iran and 13,006 global so they'd probably consider adding a rule if I submit an issue.




> * additional supplement with recommended check "trusted filter" in AdGuard subscribe custom list mechanism

I've proposed adding **PersianBlocker** to AdGuard but so far little interest has been shown from the AG team after 10 days.
`https://github.com/AdguardTeam/FiltersRegistry/issues/617`
